diff --git a/include/mbedtls/mbedtls_config.h b/include/mbedtls/mbedtls_config.h
index 1dd55c3..f1b37dd 100644
--- a/include/mbedtls/mbedtls_config.h
+++ b/include/mbedtls/mbedtls_config.h
@@ -1754,7 +1754,7 @@
  *      MBEDTLS_TLS_ECDHE_ECDSA_WITH_NULL_SHA
  *      MBEDTLS_TLS_ECDH_ECDSA_WITH_NULL_SHA
  */
-#define MBEDTLS_SSL_PROTO_TLS1_3
+//#define MBEDTLS_SSL_PROTO_TLS1_3
 
 /**
  * rief Enable support for DTLS 1.2
